### ISSUE TITLE
[BUG] likwid-bench: stream size number only up to 2^31

### ISSUE THREAD

**Issue by aw32** [RRZE-HPC/likwid#656]
In likwid-bench one can only specify the stream size up to 2^31.
Afterwards, the following message is printed:
```
Stream size cannot be read, should look like <domain>:<size>
```
If one specify exact Byte numbers, one can only go up to around 2GB.

**Comment by TomTheBear** [comment]
True, this is an issue by using only an `int` for the stream size argument. We should use a 64bit integer for it.

As written already in your other issue, we are rewriting `likwid-bench`. This does not mean that changes to the current `likwid-bench` are not accepted.

**Comment by TomTheBear** [comment]
Thanks for the PR. I merged it.
